### PR TITLE
Update to Tauri v2 (alpha)

### PR DESCRIPTION
**Pull request by nwesterhausen**
fix(build): update `tsconfig.json`
fix(deps): use `@rollup/plugin-terser`
feat: use `sys_locale` to get locale
docs: update install instructions

- Updated package versions in `package.json` per any warnings; updated to `tauri` v2 package
- Updated `tsconfig.json` to remove a warning about Promises by updating target.
- Added `moduleResolution="node"` to improve code intellisense.
- Added `exclude=["rollup.config.js"]` to remove warning when it tries to compile that file (vscode intellisense)
- Updated dependency version in `cargo.toml`
- Updated the cargo install instructions to use git and branch
- Added an example of using `dotenvy_macro` to include `APTABASE_KEY` without including it directly in source code

### CODE REVIEW

File: src/sys.rs [hunk 1]
```diff
@@ -27,7 +26,7 @@ pub struct SystemProperties {
 
 pub fn get_info() -> SystemProperties {
     let info = os_info::get();
-    let locale = locale().unwrap_or_default();
+    let locale = sys_locale::get_locale().unwrap_or(String::from("unavailable")); // locale() not available YET in tauri v2
```
Comment (goenning) [inline src/sys.rs:29]: can we keep unwrap_or_default here as well?

File: pnpm-lock.yaml
Comment (goenning) [inline pnpm-lock.yaml:1]: i'm trying to keep all projects using npm instead

Comment (nwesterhausen) [reply pnpm-lock.yaml:1]: Included by mistake, your base repo doesn't have any lock file so I actually wanted to not include one. I'll get rid of it.

File: src/sys.rs (hunk 1 above)
Comment (nwesterhausen) [reply src/sys.rs:29]: yes

**Pull request merged**